packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -185,7 +192,7 @@ const makeRenderer$ = (renderConfig: {
 	middleware: Array<Function>,
 	scripts: Array<string>,
 	enableServiceWorker: boolean,
-	cssLinks: ?Array<string>,
+	cssLinks: ?(Array<string> | Function),
```

Comment:
usually it's good to avoid `Function` types since they're not very specific. Try

```js
?MWPState => Array<string>
```